services/minio/service.go

```diff
@@ -15,7 +15,7 @@ func (s *Service) create(ctx context.Context, name string, opt pairServiceCreate
 	if err != nil {
 		return nil, err
 	}
-	err = s.service.MakeBucket(ctx, name, minio.MakeBucketOptions{})
+	err = s.service.MakeBucket(ctx, name, minio.MakeBucketOptions{Region: opt.Location})
```

Comment:
Maybe we also need to set `f.Location = opt.Location` above so that we can use `f.Location` in `utils.go`?